app/src/main/java/chat/rocket/android/chatrooms/presentation/ChatRoomsView.kt

```diff
@@ -12,4 +12,9 @@ interface ChatRoomsView : LoadingView, MessageView {
      * @param dataSet The data set to show.
      */
     suspend fun updateChatRooms(newDataSet: List<ChatRoom>)
+
+    /**
+     * User has successfully logged out from the current server.
+     */
+    fun onLogOut()
```

Comment:
You could define `onLogout()` instead of `onLogOut()` like was declared to the function `logout` in ChatRoomsPresenter class